sdk/keyvault/azure-security-keyvault-keys/test/perf/CMakeLists.txt

```diff
@@ -9,12 +9,12 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 set(
   AZURE_KEYVAULT_KEY_PERF_TEST_HEADER
-  inc/azure/keyvault/keys/test/get_key.hpp
+  inc/azure/keyvault/keys/test/get_key_test.hpp
```

Comment:
...`_perf_test`?

Reply:
And also, same question, do we need `inc/azure/...` here, or can we just put in under `inc`?